tests/TestCase/Validation/ValidationTest.php

```diff
@@ -3103,4 +3103,18 @@ public function testImageWidth()
         $this->assertFalse(Validation::imageWidth($upload, '>', 2000));
         $this->assertFalse(Validation::imageWidth($upload, '==', 3000));
     }
+
+    /**
+     * Test color
+     */
+    public function testColor()
+    {
+        $this->assertTrue(Validation::color('#F01234'));
+        $this->assertTrue(Validation::color('#F56789'));
+        $this->assertTrue(Validation::color('#abcdef'));
+        $this->assertTrue(Validation::color('#ABCDEF'));
+
+        $this->assertFalse(Validation::color('#fff'));
+        $this->assertFalse(Validation::color('ffffff'));
```

Comment:
Thank you. I would like to choose `hexColor()`. It represents well what this validation does.